src/org/zaproxy/zap/extension/ascanrulesBeta/ExtensionAscanRulesBeta.java

```diff
@@ -61,27 +53,4 @@ public boolean canUnload() {
 		return true;
 	}
 
-	@Override
-	public void unload() {
```

Comment:
It would be great if it removed the callback from the extension (otherwise it will not be fully unloaded).